packages/ibm-products/src/components/HTTPErrors/HTTPErrorOther/HTTPErrorOther.tsx

```diff
@@ -21,13 +21,54 @@ import { pkg } from '../../../settings';
 const blockClass = `${pkg.prefix}--http-errors`;
 const componentName = 'HTTPErrorOther';
 
+interface HTTPErrorOtherProps extends PropsWithChildren {
+  /**
+   * Provide an optional class to be applied to the containing node
+   */
+  className?: string;
+  /**
+   * String that will provide the description for the HTTP error code
+   */
+  description: string;
+  /**
+   * String that will describe the error that occurred
+   */
+  errorCodeLabel: string;
+  /**
+   * Links that will display for extra context when receiving particular errors
+   */
+  links: Array<{
```

Comment:
`links` should be optional here
```suggestion
  links?: Array<{
```